### DISCUSSION TITLE
He3_windows_installer_x64_1.3.2.exe nothing to show

### DISCUSSION THREAD

**Discussion by lx19990999** [he3-app/he3-feedback#145] [Bug Report]
I update from version 1.2 ,  but it has nothing. I reinstall it , but it always show this.
he3.app , the website also can not download old version.
my system: win11x64 22h2 22621.1555
![He3_E4eT1DRoij](https://user-images.githubusercontent.com/16575950/233245576-3d2fb1c7-5852-48d8-8906-19b995db1bb7.png)

**Comment by lyzhang1999** [comment]
Can you leave your email? We will contact you.

**Reply by lx19990999** [reply]
bHgxOTk5MDk5OUBnbWFpbC5jb20=

**Comment by lyzhang1999** [comment]
@zhangze-github

**Comment by zhangze-github** [comment]
We are checking the exception, please try to restore factory settings first
![image](https://user-images.githubusercontent.com/41512027/233248690-f344aa95-97bb-450e-891c-7323a5fa96a7.png)
![image](https://user-images.githubusercontent.com/41512027/233248728-04d00f6e-bf30-42f7-8f82-b284ce10193d.png)

**Reply by lx19990999** [reply]
it not useful

**Comment by zhangze-github** [comment]
There may be a local cache of dirty data，Please try to clear the cache and reinstall the App.
The file directory is： C:\Users\<you>\AppData\Roaming\he3 ，please delete it.

Another possibility is a network problem, please try to visit the website to see if it works.
https://portal.he3app.com/

Thank you and look forward to your feedback.

**Reply by lx19990999** [reply]
yes, it is a network problem. I use fiddler to catch url
and I use proxy global, it is show.

PS C:\Users\Administrator> ping app.posthog.com

正在 Ping posthog-cloud-ingress-alb-918737860.us-east-1.elb.amazonaws.com [34.193.114.220] 具有 32 字节的数据:
请求超时。
请求超时。
请求超时。
请求超时。

34.193.114.220 的 Ping 统计信息:
    数据包: 已发送 = 4，已接收 = 0，丢失 = 4 (100% 丢失)，
PS C:\Users\Administrator> ping api.he3app.com

正在 Ping joic4fbt.slt.sched.tdnsv8.com [180.95.234.190] 具有 32 字节的数据:
来自 180.95.234.190 的回复: 字节=32 时间=61ms TTL=52
来自 180.95.234.190 的回复: 字节=32 时间=89ms TTL=52
来自 180.95.234.190 的回复: 字节=32 时间=51ms TTL=52
来自 180.95.234.190 的回复: 字节=32 时间=74ms TTL=52

180.95.234.190 的 Ping 统计信息:
    数据包: 已发送 = 4，已接收 = 4，丢失 = 0 (0% 丢失)，
往返行程的估计时间(以毫秒为单位):
    最短 = 51ms，最长 = 89ms，平均 = 68ms
PS C:\Users\Administrator> ping he3-1309519128.cos.accelerate.myqcloud.com

正在 Ping cos.accelerate.myqcloud.com [111.202.29.45] 具有 32 字节的数据:
来自 111.202.29.45 的回复: 字节=32 时间=113ms TTL=50
来自 111.202.29.45 的回复: 字节=32 时间=79ms TTL=50
来自 111.202.29.45 的回复: 字节=32 时间=72ms TTL=50
来自 111.202.29.45 的回复: 字节=32 时间=92ms TTL=50

111.202.29.45 的 Ping 统计信息:
    数据包: 已发送 = 4，已接收 = 4，丢失 = 0 (0% 丢失)，
往返行程的估计时间(以毫秒为单位):
    最短 = 72ms，最长 = 113ms，平均 = 89ms
PS C:\Users\Administrator> ping o4504088220925952.ingest.sentry.io

正在 Ping o4504088220925952.ingest.sentry.io [34.120.195.249] 具有 32 字节的数据:
来自 34.120.195.249 的回复: 字节=32 时间=226ms TTL=111
请求超时。
来自 34.120.195.249 的回复: 字节=32 时间=200ms TTL=111
请求超时。

34.120.195.249 的 Ping 统计信息:
    数据包: 已发送 = 4，已接收 = 2，丢失 = 2 (50% 丢失)，
往返行程的估计时间(以毫秒为单位):
    最短 = 200ms，最长 = 226ms，平均 = 213ms
